net/rpmsg/rpmsg_sockif.c

```diff
@@ -386,7 +386,14 @@ static int rpmsg_socket_ept_cb(FAR struct rpmsg_endpoint *ept,
 static inline void rpmsg_socket_destroy_ept(
                     FAR struct rpmsg_socket_conn_s *conn)
 {
-  if (conn && conn->ept.rdev)
+  if (!conn)
```

Comment:
```suggestion
  if (conn == NULL)
```

Reply:
If the coding guideline say this, show me.

Reply:
Good point. We should update the CS.

It is a readability issue

```
find . -type f | xargs grep -E '\(\!' | wc -l
4875
find . -type f | xargs grep -E '\(.* .= NULL' | wc -l
6786
```

Reply:
If this you should update the guideline first, and add the rules to checkpatch.
For this time, I will follow old rules.
So this is not one issue, and don't need to change my patch for this time.

Reply:
@davids5 there are many places which check the pointer by if (p) or if (!p), so please send the suggestion to the mail list and get approved before we enforce your personal code style rule.